### ISSUE TITLE
Installation on Ubuntu also requires `pyhton3-dev` package

### ISSUE THREAD

**Issue by ahorn42** [openvmp/partcad#200]
Hi,

I just tried to install `partcad-cli` and noticed the hint about `libcairo2-dev`. Even if I had that installed it failed with:

```
      Run-time dependency cairo found: YES 1.18.0
      Run-time dependency python found: NO (tried pkgconfig, pkgconfig and sysconfig)
      
      ../cairo/meson.build:51:15: ERROR: Python dependency not found
```

I also needed to install `python3-dev` to be able to install it. Maybe this should be added to the readme as well.

best regards!

**Comment by openvmp** [comment]
Could you, please, confirm whether 'python3-dev' completely solved the problem or not?

**Comment by ahorn42** [comment]
I think yes.

First I only installed `libcairo2-dev` as stated in the readme. Then I experienced the error from above. I then installed `python3-dev` and I could run the installation successfully. I did not install or change anything else than `python3-dev`.

The installation took place on an Ubuntu 24.04.1 LTS

**Comment by openvmp** [comment]
Fixed in #203

**Comment by openvmp** [comment]
@ahorn42 , thank you!
